test/index.ts

```diff
@@ -269,68 +275,71 @@ describe("Testing Toucan-SDK", function () {
 
   describe("Testing subgraph related methods", function () {
     it("Should fetch user batches", async function () {
-      expect((res = await toucan.fetchUserBatches(addr1.address))).to.not.throw;
+      expect((res = await subgraph.fetchUserBatches(addr1.address))).to.not
+        .throw;
     });
 
     describe("Testing TCO2 Token fetching methods", function () {
       it("Should fetch details about TCO2 based on its address", async function () {
         expect(
-          (res = await toucan.fetchTCO2TokenById(
+          (res = await subgraph.fetchTCO2TokenById(
             "0x0044c5a5a6f626b673224a3c0d71e851ad3d5153"
           ))
         ).to.not.throw;
       });
 
       it("Should fetch details about TCO2 based on its full symbol", async function () {
```

Comment:
just a side-note: the test says that it should fetch details but all you're checking is that it doesn't throw. that's not quite the same